curriculum/challenges/english/25-front-end-development/quiz-css-positioning/66ed8ffcf45ce3ece4053eb5.md

```diff
@@ -628,3 +628,494 @@ Which of the following is the default value of the `position` property?
+
+---
+
+`position: static`
+
+#### --answer--
+
+`transform` 
+
+### --question--
+
+#### --text--
+
+Which ancestor does an absolutely positioned element use as its containing block?
+
+#### --distractors--
+
+`The root <html> element`
+
+---
+
+`The viewport`
+
+---
+
+`The first element with display: block`
+
+#### --answer--
+
+`The nearest ancestor whose position is not static` 
+
+### --question--
+
+#### --text--
+
+Which of these is ignored by position: fixed when determining its containing block?
+
+#### --distractors--
+
+`The <body> element`
+
+---
+
+`A parent with position: relative`
+
+---
+
+`The root element`
+
+#### --answer--
+
+`All ancestors—position: fixed elements are always relative to the viewport` 
+
+### --question--
+
+#### --text--
+
+If two positioned elements overlap without any z-index specified, what determines which one paints on top?
+
+#### --distractors--
+
+`The element with larger width`
+
+---
+
+`Alphabetical order of their class names`
+
+---
+
+`Their position value (absolute over relative)`
+
+#### --answer--
+
+`Their order in the HTML (later elements paint on top)` 
+
+### --question--
+
+#### --text--
+
+Which CSS snippet correctly centers an absolutely positioned box both horizontally and vertically in its container?
+
+#### --distractors--
+
+```css
+.box {
+  position: absolute;
+  margin: auto;
+  width: 50%;
+  height: 50%;
+}
+```
+
+---
+
+```css
+.box {
+  position: absolute;
+  right: 50%;
+  bottom: 50%;
+  transform: translate(50%, 50%);
+}
+```
+
+---
+
+```css
+.box {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+```
+
+#### --answer--
+
+```css
+.box {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+```
+
+### --question--
+
+#### --text--
+
+What does float:right do to an element?
+
+#### --distractors--
+
+`Positions the element at the bottom of its container`
+
+---
+
+`Removes the element from the normal flow and sticks it to the right side of the viewport`
+
+---
+
+`Centers the element horizontally`
+
+#### --answer--
+
+`Places the element on the right side of its container, allowing inline content to wrap around it` 
+
+### --question--
+
+#### --text--
+
+Which values can the clear property take to clear both left and right floats?
+
+#### --distractors--
+
+`clear: all-sides;`
+
+---
+
+`clear: both-sides;`
+
+---
+
+`clear: left-right;`
+
+#### --answer--
+
+`Places the element on the right side of its container, allowing inline content to wrap around it` 
+
+### --question--
+
+#### --text--
+
+What happens when you apply `float: left;` to an element in CSS?
+
+#### --distractors--
+
+`It centers the element horizontally inside its container`  
+
+---
+
+`It aligns the element at the bottom of the container`  
+
+---
+
+`It moves the element to the right side of the container`
+
+#### --answer--
+
+`It removes the element from normal flow and positions it to the left, allowing text to wrap around it`
```

Comment:
these don't need to be wrapped in backticks. please fix.
plus they need periods.